kvbc/tools/db_integrity_check/integrity_checker.cpp

```diff
@@ -219,15 +223,22 @@ concord::kvbc::categorization::RawBlock IntegrityChecker::getBlock(const BlockId
   return kvbc::categorization::RawBlock::deserialize(rawBlockSer);
 }
 
-void IntegrityChecker::validateAll() const {
+Digest IntegrityChecker::computeBlockDigest(const BlockId& block_id, const std::string_view& block) const {
```

Comment:
Pass string view by value:
https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/

Reply:
Yes, I thought about it, it makes more sense, just like a shared_pointer  :-)

Reply:
Did you change it?

Reply:
I'll do in successive commit coming shortly